devdoc/CodeSubmissions.md

```diff
@@ -6,7 +6,19 @@ We welcome submissions to the platform especially in the form of pull requests i
 There are a few things that should be considered when creating a PR to increase the likelihood that they can be accepted quickly.
 
 * Write a good commit message\
-  The format should be HPCC-XXXXX (where XXXXX is the bug number) followed by a description of the issue.  The text should make sense in a change log by itself - without reference to the jira or the contents of the PR.  We should aim to increase the information that is included as part of the commit message - not rely on on the jira.
+  The format should be HPCC-XXXXX (where XXXXX is the bug number) followed by a description of the issue.  The text should make sense in a change log by itself - without reference to the jira or the contents of the PR.  We should aim to increase the information that is included as part of the commit message - not rely on the jira.
+* Separate the JIRA and commit message with a space.
+  - HPCC-XXXXX This is the description of the solution
```

Comment:
The leading '-' in - HPCC-XXXXX This is the description of the solution is not meant to be included in the commit message.  A little misleading formatting issue